gen/src/generator.toml

```diff
@@ -2,4 +2,4 @@
 library_name = "libmpi"
 output_common_file_path = "./out/common.jl"
 output_api_file_path = "./out/api.jl"
-output_ignorelist = ["^PMPI_", "^QMPI_", "^MPIX_", "^PMPIX_", "^QMPIX_", "^MPI_T_", "^MPI_Session_", "MPI_Status_f082c", "MPI_Status_f082f", "MPI_Status_f2f08", "MPI_Status_c2f08", "MPI_Group_from_session_pset"]
+output_ignorelist = ["^PMPI_", "^QMPI_", "^MPIX_", "^MPIR_", "^PMPIX_", "^QMPIX_", "^MPI_T_", "^MPI_Session_", "MPI_Status_f082c", "MPI_Status_f082f", "MPI_Status_f2f08", "MPI_Status_c2f08", "MPI_Group_from_session_pset"]
```

Comment:
I noticed that there are a lot of functions called `*_c` which look redundant and aren't available in OpenMPI. From MPICH manpage I can't tell the difference with the non-`*_c` ones. You may want to skip also those

Reply:
You're right. I've also improved the regex for other filters.

Reply:
The _c ones are the "big count" ones added in MPI 4. They allow 64-bit length buffers

Reply:
Oh, then I guess we shall restore them ?

Reply:
Oh, wow, MPICH manpages are highly non descriptive: see for example https://www.mpich.org/static/docs/latest/www3/MPI_Accumulate_c.html vs https://www.mpich.org/static/docs/latest/www3/MPI_Accumulate.html.  The difference is the `_c` in the function name and the arguments types repeated twice, you can't tell anything else from that.

Reply:
> arguments types repeated twice

They could be more descriptive, but `int` is replaced by `MPI_Count`.